cranelift-codegen/meta/src/cdsl/type_inference.rs

```diff
@@ -364,7 +364,6 @@ impl TypeEnvironment {
 
             // Sanity check: translated constraints should refer only to real variables.
             for arg in constraint.typevar_args() {
-                assert!(vars_tv.contains(arg));
```

Comment:
Why was this assert removed?

Reply:
@bnjbvr? (he made that commit)

Reply:
@bjorn3 This has landed separately in another PR, see [commit message](https://github.com/CraneStation/cranelift/commit/137b13815372c0b61b8de42e7b481a2a8f8b10bd).